advocacy_docs/partner_docs/VisionXGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for VisionX'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      | SIB Visions       |
+| **Partner Product**   |  VisionX       |
+| **Web Site**      |    https://visionx.sibvisions.com/   |
+| **Version & Platform**   |  VisionX 5.6.925 : CentOS7      |
+| **Product Description**      |  VisionX is a low-code platform for creating web and mobile applications extremely quickly and easily. VisionX low-code platform is also a good replacement for Oracle's APEX low-code platform in the Postgres world.     |
```

Comment:
I could go with this as the Description until we get feedback from Roland.

VisionX is an open and flexible low-code platform that enables businesses and IT users to build web and mobile applications themselves.  VisionX can also help you transform Oracle Forms systems within months instead of years.  By combining template/database/ FMB-based screen generation, fast visual development, and Oracle Forms features, you can build your own systems quickly and efficiently.